File: Arreglos/Java/Filas_DI_I.java
public class Filas_DI_I{
    public static void main(String[] ars){

        int DIMENSION = 5;
        int Matriz[][] = new int[DIMENSION][DIMENSION];

        for(int Filas = 0; Filas < DIMENSION; ++Filas){
            for(int Columnas = 0; Columnas <= Filas; ++Columnas){
                                
                System.out.print(Filas+","+Columnas+" ");

            }

            System.out.println();

        }

    }
}

File: Arreglos/C/Filas_DI_I.c
#define DIMENSION 5

#include <stdio.h>

int main() {
    int arreglo[DIMENSION][DIMENSION];
        int filas = 0, columnas = 0;

        for (filas = 0; filas < DIMENSION; filas++) {

            for (columnas = 0; columnas < DIMENSION; columnas++) {

                if (columnas + filas >= DIMENSION - 1) {
                    printf("(%d,%d) ", filas, columnas);
                }
                else{
                    printf("      ");
                }
            }

            printf("\n");
        }
    return 0;
}